main.go

```diff
@@ -1,92 +1,37 @@
-// This is the name of our package
-// Everything with this package name can see everything
-// else inside the same package, regardless of the file they are in
 package main
 
-// These are the libraries we are going to use
-// Both "fmt" and "net" are part of the Go standard library
 import (
-	// "fmt" has methods for formatted I/O operations (like printing to the console)
-	"fmt"
-	// The "net/http" library has methods to implement HTTP clients and servers
-	"net/http"
-  "github.com/gorilla/mux"
 	"database/sql"
-	"encoding/json"
 	"log"
-	_ "github.com/lib/pq"
-	"io/ioutil"
+	"net/http"
 
-	"models"
+	"github.com/gorilla/mux"
+	_ "github.com/lib/pq"
 )
 
-var db *sql.DB
-
-// The new router function creates the router and
-// returns it to us. We can now use this function
-// to instantiate and test the router outside of the main function
 func newRouter() *mux.Router {
-	// "Signin" and "Signup" are handler that we will implement
-	// initialize our database connection
 	r := mux.NewRouter()
-	r.HandleFunc("/login", login).Methods("GET")
-	r.HandleFunc("/signup", signup).Methods("POST")
+	r.HandleFunc("/login", loginHandler).Methods("POST")
+	r.HandleFunc("/signup", signUpHandler).Methods("POST")
+	r.HandleFunc("/welcome", welcomeHandler).Methods("GET")
 	return r
 }
 
+var db *sql.DB
+var jwtKey = []byte("secret_key")
```

Comment:
A way to handle these type of things better:

- Get the value from a file that is not included in version control. ``(IE: .env,  secretts, etc...)``
- Include steps to generate the file and value in README file.

Not sure how the first point plays with the strict importining system in golang. We'll see!